helm-chart/pangeo-binder/values.yaml

```diff
@@ -2,6 +2,58 @@ binderhub:
   rbac:
     enabled: true
 
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: nginx
+    https:
+      enabled: true
+      type: kube-lego
+
   jupyterhub:
     singleuser:
       serviceAccountName: daskkubernetes
+    ingress:
+      enabled: true
+      annotations:
+        ingress.kubernetes.io/proxy-body-size: 64m
+        kubernetes.io/ingress.class: nginx
+        kubernetes.io/tls-acme: 'true'
+
+nginx-ingress:
+  rbac:
+    create: true
+  defaultBackend:
+    minAvailable: 0
+  # statsExporter:
+  #   service:
+  #     annotations:
+  #       prometheus.io/scrape: 'true'
+  #       prometheus.io/port: '10254'
```

Comment:
will we need to turn this on for the full deployment? What would happen if we uncommented these lines?

Reply:
I don't think we have the stats service set up to scrape these stats. I don't think there's any harm in turning this on, but I don't think we'll see anything from it either.